packages/design-system/src/icons/clear-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="7" height="7" viewBox="0 0 7 7" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Try to use `fill="currentColor"` and give color by "color" style attribute. This way we can leverage Tailwinds design system bindings to color match.

Reply:
Yes, this is being implemented at places as and when a requirement for reusability occurs with different colors for SVG.